core-services/prow/02_config/_plugins.yaml

```diff
@@ -520,6 +520,12 @@ config_updater:
         - ci
       gzip: true
       name: ci-operator-misc-configs
+    ci-operator/config/coreos/**/*-rhcos*.yaml:
```

Comment:
I only barely understand this file, but offhand I'm not sure why we wouldn't just include all YAML files (i.e. `ci-operator/config/coreos/**/*.yaml`). And similarly below.

Reply:
@jlebon I was trying to not mess or touch other things I didn't add in the configs/jobs hehe 🙃. It shouldn't be a problem though.